astropy/table/table.py

```diff
@@ -3857,6 +3857,13 @@ class QTable(Table):
     except that columns with an associated ``unit`` attribute are converted to
     `~astropy.units.Quantity` objects.
 
+    .. note:: When a column of ``int`` ``dtype`` is converted to ``Quantity``, its
```

Comment:
The standard for multi-line directives like this is to put the note on a different line

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-note

```suggestion
    .. note::

       When a column of ``int`` ``dtype`` is converted to ``Quantity``, its
```

Reply:
Resolved 👍